dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -341,18 +406,14 @@ public void ContentFallbackFalse_PageFallbackTrue_PageEnglish_ViewEnglishContent
      *
      */
     @Test
-    public void ContentFallbackFalse_PageFallbackTrue_PageEnglishAndSpanish_ViewEnglishContent1And2_ViewSpanishContent2And3() throws Exception{
+    @UseDataProvider("cases")
+    public void ContentFallbackFalse_PageFallbackTrue_PageEnglishAndSpanish_ViewEnglishContent1And2_ViewSpanishContent2And3(final Container container, final Template template) throws Exception{
```

Comment:
I would add javadoc explaining better the given scenario and expected result

Reply:
this test is not new